tools/refresh-aws-pro-ids

```diff
@@ -32,40 +33,51 @@ def main():
     os.chdir("products")
     aws_ids = {}
     for aws_listing in glob.glob("listing-aws-pro-*"):
-        m = re.match(
-            r"^listing-aws-pro-(?P<release>\w+).yaml$", aws_listing
-        )
+        m = re.match(r"^listing-aws-pro-(?P<release>\w+).yaml$", aws_listing)
```

Comment:
I don't know if this is feasible, but maybe we should try to update the pro fips images too

Reply:
Lemme check it out

Reply:
I will have to remove the focal stuff